amr-wind/incflo_advance.cpp

```diff
@@ -47,15 +47,20 @@ void incflo::pre_advance_stage2()
  *
  * \callgraph
  */
-void incflo::advance()
+void incflo::advance(const int ifixed_point_iteration)
 {
     BL_PROFILE("amr-wind::incflo::advance");
+    if (ifixed_point_iteration == 0) {
+        m_sim.pde_manager().advance_states();
+    }
 
-    m_sim.pde_manager().advance_states();
-
-    ApplyPredictor();
+    ApplyPredictor(false, ifixed_point_iteration);
 
     if (!m_use_godunov) {
+        AMREX_ALWAYS_ASSERT_WITH_MESSAGE(
+            ifixed_point_iteration == 0,
+            "Advection iterations are not supported for MOL");
```

Comment:
The error message needs to be adjusted now that we call this fixed point iterations